src/stream_base.cc

```diff
@@ -335,7 +335,7 @@ int StreamBase::WriteString(const FunctionCallbackInfo<Value>& args) {
     }
   }
 
-  StreamWriteResult res = Write(&buf, 1, send_handle, req_wrap_obj);
+  StreamWriteResult res = Write(&buf, 1, send_handle, req_wrap_obj, try_write);
```

Comment:
@mojodna `skip_try_write` is used here. The above `DoTryWrite()` yields a partial write, so here, we tell the `Write()` do not bother trying a synchronous write, which is `DoTryWrite()`, just perform an asynchronous write directly, which is `DoWrite()`. Let's say the underlying stream is a socket, with this little change, we save one syscall(`write(1)`) in case the socket is busy.

Reply:
thx, I did not spot this.